service/controller/app/resource/chart/desired.go

```diff
@@ -74,16 +74,19 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 
 	annotations := generateAnnotations(cr.GetAnnotations())
 
-	if key.InCluster(cr) {
```

Comment:
Previously we only put webhook annotation if it's an in-cluster object. Not anymore.